### PR TITLE
Add 4 more authors

### PR DESCRIPTION
**Pull request by certik**
@mattpap, @scolobb, @flacjacket, can you please review and update your author information?

### CODE REVIEW

File: authors/authors.json [hunk 1]
```diff
@@ -8,6 +8,14 @@
         "sympy_commits": 2636
     },
     {
+        "email": "mattpap@gmail.com",
+        "github_id": "mattpap",
+        "institution": "Continuum Analytics",
```
Comment (mattpap) [inline authors/authors.json]: @certik, I'm not sure if it matters, but in formal documents we use `Continuum Analytics, Inc.`.

Comment (certik) [inline authors/authors.json]: Fixed in 6fe6927dd9273dbfa5e8973a3934eb96e46403c6.

Comment (mattpap) [inline authors/authors.json]: Thanks.

File: authors/authors.json [hunk 2]
```diff
@@ -48,6 +64,14 @@
         "sympy_commits": 186
     },
     {
+        "email": "sean.v.775@gmail.com",
+        "github_id": "flacjacket",
+        "institution": "??",
+        "institution_address": "??",
```
Comment (flacjacket) [inline authors/authors.json]: University of Illinois at Urbana-Champaign
1110 W Green St, Urbana, IL 61801, USA

Comment (certik) [inline authors/authors.json]: Thanks, updated in e509836709bf18d83871f0e16efd8a44692eb47d.

File: authors/authors.json [hunk 3]
```diff
@@ -16,6 +25,15 @@
         "sympy_commits": 1443
     },
     {
+        "email": "mrocklin@gmail.com",
+        "github_id": "mrocklin",
+        "institution": "Continuum Analytics, Inc.",
+        "institution_address": "221 W. 6th St. #1550, Austin, TX 78701 USA",
+        "institution_address_siam": "Austin, TX 78701",
```
Comment (sumith1896) [inline authors/authors.json:34]: @mrocklin @certik Could you verify this for me? I'm unsure here

Comment (mrocklin) [inline authors/authors.json:34]: Seems fine to me.

Comment (sumith1896) [inline authors/authors.json:34]: Great!

Comment (certik) [inline authors/authors.json:34]: Yes, that's correct. @Sumith1896, thanks for adding @mrocklin in.

**Comment by certik** [comment]
This is ready to go in if tests pass.

File: authors/authors.json [hunk 4]
```diff
@@ -24,6 +32,14 @@
         "sympy_commits": 263
     },
     {
+        "email": "scolobb@colimite.fr",
+        "github_id": "scolobb",
+        "institution": "??",
+        "institution_address": "??",
+        "name": "Sergiu Ivanov",
+        "sympy_commits": 260
+    },
```
Comment (scolobb) [inline authors/authors.json]: Officially, my affiliation goes as follows:

``` python
{
        "email": "sergiu.ivanov@u-pec.fr",
        "github_id": "scolobb",
        "institution": "Université Paris Est Créteil",
        "institution_address": "61 av. Général de Gaulle\n94010 Créteil, France",
        "name": "Sergiu Ivanov",
        "sympy_commits": 260
}
```

Comment (certik) [inline authors/authors.json]: Thanks @scolobb, I didn't know when you will update it, so I separate your update from the rest into #152.

**Comment by certik** [comment]
As of 8eb076fd2f4804d91ed964d18b2cbd83a3d9de87:

# Article Authors

All authors satisfy the following ICMJE [criteria](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html#two) for authorship:
1. Substantial contributions to the conception or design of the work; or the
   acquisition, analysis, or interpretation of data for the work; AND
2. Drafting the work or revising it critically for important intellectual
   content; AND
3. Final approval of the version to be published; AND
4. Agreement to be accountable for all aspects of the work in ensuring that
   questions related to the accuracy or integrity of any part of the work are
   appropriately investigated and resolved.

The column `SymPy Commits` lists how many commits the authors contributed to
SymPy (with a link to the actual commits), which shows how the requirement 1.
is satisfied.

The column `Article Contributions` lists all commits and comments to the
article itself, which shows how the requirement 2. is satisfied.

All authors approved the final version of the article, satisfying the
requirement 3.

Finally, all authors agreed to the requirement 4.

| Author | GitHub Profile | SymPy Commits | Article Contributions | Affiliation |
| :-- | :-- | :-: | :-- | :-- |
| Aaron Meurer | [@asmeurer](https://github.com/asmeurer) | [2636](https://github.com/sympy/sympy/commits?author=asmeurer) | [commits](https://github.com/sympy/sympy-paper/commits?author=asmeurer), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:asmeurer&type=Issues) | University of South Carolina, Columbia, SC 29201 |
| Mateusz Paprocki | [@mattpap](https://github.com/mattpap) | [1450](https://github.com/sympy/sympy/commits?author=mattpap) | [commits](https://github.com/sympy/sympy-paper/commits?author=mattpap), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:mattpap&type=Issues) | Continuum Analytics, Inc., Austin, TX 78701 |
| Ondřej Čertík | [@certik](https://github.com/certik) | [1443](https://github.com/sympy/sympy/commits?author=certik) | [commits](https://github.com/sympy/sympy-paper/commits?author=certik), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:certik&type=Issues) | Los Alamos National Laboratory, Los Alamos, NM 87545 |
| Matthew Rocklin | [@mrocklin](https://github.com/mrocklin) | [1046](https://github.com/sympy/sympy/commits?author=mrocklin) | [commits](https://github.com/sympy/sympy-paper/commits?author=mrocklin), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:mrocklin&type=Issues) | Continuum Analytics, Inc., Austin, TX 78701 |
| AMiT Kumar | [@aktech](https://github.com/aktech) | [263](https://github.com/sympy/sympy/commits?author=aktech) | [commits](https://github.com/sympy/sympy-paper/commits?author=aktech), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:aktech&type=Issues) | Delhi Technological University, Shahbad Daulatpur, Bawana Road, New Delhi 110042, India |
| Jason K. Moore | [@moorepants](https://github.com/moorepants) | [253](https://github.com/sympy/sympy/commits?author=moorepants) | [commits](https://github.com/sympy/sympy-paper/commits?author=moorepants), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:moorepants&type=Issues) | University of California, Davis, Davis, CA 95616 |
| Sartaj Singh | [@leosartaj](https://github.com/leosartaj) | [199](https://github.com/sympy/sympy/commits?author=leosartaj) | [commits](https://github.com/sympy/sympy-paper/commits?author=leosartaj), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:leosartaj&type=Issues) | Indian Institute of Technology (BHU), Varanasi, Uttar Pradesh 221005, India |
| Thilina Rathnayake | [@thilinarmtb](https://github.com/thilinarmtb) | [186](https://github.com/sympy/sympy/commits?author=thilinarmtb) | [commits](https://github.com/sympy/sympy-paper/commits?author=thilinarmtb), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:thilinarmtb&type=Issues) | University of Moratuwa, Bandaranayake Mawatha, Katubedda, Moratuwa 10400, Sri Lanka |
| Sean Vig | [@flacjacket](https://github.com/flacjacket) | [186](https://github.com/sympy/sympy/commits?author=flacjacket) | [commits](https://github.com/sympy/sympy-paper/commits?author=flacjacket), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:flacjacket&type=Issues) | University of Illinois at Urbana-Champaign, Urbana, IL 61801 |
| Brian E. Granger | [@ellisonbg](https://github.com/ellisonbg) | [160](https://github.com/sympy/sympy/commits?author=ellisonbg) | [commits](https://github.com/sympy/sympy-paper/commits?author=ellisonbg), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:ellisonbg&type=Issues) | California Polytechnic State University, San Luis Obispo, CA 93407 |
| Richard P. Muller | [@rpmuller](https://github.com/rpmuller) | [158](https://github.com/sympy/sympy/commits?author=rpmuller) | [commits](https://github.com/sympy/sympy-paper/commits?author=rpmuller), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:rpmuller&type=Issues) | Center for Computing Research, Sandia National Laboratories, Albuquerque, NM 87185 |
| Francesco Bonazzi | [@Upabjojr](https://github.com/Upabjojr) | [150](https://github.com/sympy/sympy/commits?author=Upabjojr) | [commits](https://github.com/sympy/sympy-paper/commits?author=Upabjojr), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:Upabjojr&type=Issues) | Max Planck Institute of Colloids and Interfaces, Am Mühlenberg 1, 14476 Potsdam, Germany |
| Harsh Gupta | [@hargup](https://github.com/hargup) | [123](https://github.com/sympy/sympy/commits?author=hargup) | [commits](https://github.com/sympy/sympy-paper/commits?author=hargup), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:hargup&type=Issues) | Indian Institute of Technology Kharagpur, Kharagpur, West Bengal 721302, India |
| Shivam Vats | [@shivamvats](https://github.com/shivamvats) | [122](https://github.com/sympy/sympy/commits?author=shivamvats) | [commits](https://github.com/sympy/sympy-paper/commits?author=shivamvats), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:shivamvats&type=Issues) | Indian Institute of Technology Kharagpur, Kharagpur, West Bengal 721302, India |
| Fredrik Johansson | [@fredrik-johansson](https://github.com/fredrik-johansson) | [119](https://github.com/sympy/sympy/commits?author=fredrik-johansson) | [commits](https://github.com/sympy/sympy-paper/commits?author=fredrik-johansson), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:fredrik-johansson&type=Issues) | INRIA Bordeaux-Sud-Ouest - LFANT project-team, 200 Avenue de la Vieille Tour, 33405 Talence, France |
| Fabian Pedregosa | [@fabianp](https://github.com/fabianp) | [114](https://github.com/sympy/sympy/commits?author=fabianp) | [commits](https://github.com/sympy/sympy-paper/commits?author=fabianp), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:fabianp&type=Issues) | INRIA - SIERRA project-team, 2 Rue Simone IFF, 75012 Paris, France |
| Ashutosh Saboo | [@ashutoshsaboo](https://github.com/ashutoshsaboo) | [17](https://github.com/sympy/sympy/commits?author=ashutoshsaboo) | [commits](https://github.com/sympy/sympy-paper/commits?author=ashutoshsaboo), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:ashutoshsaboo&type=Issues) | Birla Institute of Technology and Science, Pilani, K.K. Birla Goa Campus, NH 17B Bypass Road, Zuarinagar, Sancoale, Goa 403726, India |
| Isuru Fernando | [@isuruf](https://github.com/isuruf) | [7](https://github.com/sympy/sympy/commits?author=isuruf) | [commits](https://github.com/sympy/sympy-paper/commits?author=isuruf), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:isuruf&type=Issues) | University of Moratuwa, Bandaranayake Mawatha, Katubedda, Moratuwa 10400, Sri Lanka |
| Sumith | [@Sumith1896](https://github.com/Sumith1896) | [6](https://github.com/sympy/sympy/commits?author=Sumith1896) | [commits](https://github.com/sympy/sympy-paper/commits?author=Sumith1896), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:Sumith1896&type=Issues) | Indian Institute of Technology Bombay, Powai, Mumbai, Maharashtra 400076, India |
| Robert Cimrman | [@rc](https://github.com/rc) | [1](https://github.com/sympy/sympy/commits?author=robert.cimrman@gmail.com) | [commits](https://github.com/sympy/sympy-paper/commits?author=rc), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:rc&type=Issues) | New Technologies - Research Centre, University of West Bohemia, Univerzitní 8, 306 14 Plzeň, Czech Republic |

Note: This file was automatically generated from `authors.json` using `list_text.py`.

**Comment by certik** [comment]
The link to @mrocklin commits to this repo is empty, but that's because his commit is only on the reviews branch. Once it is merged into master, it will appear there.

**Comment by certik** [comment]
Based on my `authors.md` table above, all three authors satisfied the authorship criteria.

**Pull request merged**